lib/priority_queue.dart

```diff
@@ -13,6 +13,16 @@ import "dart:collection" show SplayTreeSet;
  */
 abstract class PriorityQueue<E> {
   /**
+   * Creates an empty [PriorityQueue].
+   *
+   * The created [PriorityQueue] is a plain [HeapPriorityQueue]. As such,
+   * [Comparable.compare] is used to compare the priority of elements.
+   *
+   * The queue is equivalent to one created by `new HeapPriorityQueue<E>()`.
+   */
+  factory PriorityQueue() = HeapPriorityQueue<E>;
```

Comment:
I'm not really sure why the default Set constructor doesn't have those arguments, actually. Regardless, the ordering of a priority queue is pretty central to its behavior, so it should be customizable in the main constructor.